_topics/en/erlay.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Erlay
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - P2P Network Protocol
+  - Bandwidth Reduction
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Erlay** is proposal to use set reconciliation between Bitcoin full
+  nodes as part of relaying unconfirmed transactions.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  The currently-used Bitcoin gossip protocol expects each node to
```

Comment:
'expects' seems like an odd word to me to use here, since it implies that a node has an expectation that all of its peers will relay a transaction to it.

Reply:
done